src/index.ts

```diff
@@ -190,6 +201,11 @@ function mouseMoveHandler(event: MouseEvent) {
                 atomMouseMove(event);
             }
             break;
+        case "dragMode":
+            if (hasMouseDown && hasMouseIn) {
+                dragMouseMove(event);
+            }
+            break;
```

Comment:
If we need to do this for all cases, we should move the if statement outside the switch statement.